includes/save.php

```diff
@@ -451,11 +451,8 @@
         } else {
             $message .= sprintf(__('This is to notify you that %s created a <a href="%s">new meeting</a> on the %s site.', '12-step-meeting-list'), $user->display_name, get_permalink($post->ID), get_bloginfo('name'));
         }
-        $message .= '</p><table style="font:14px arial;width:100%;border-collapse:collapse;padding:0;">';
-        $fields = array_merge(
-            ['name', 'day', 'time', 'end_time', 'types', 'notes', 'location', 'formatted_address', 'region', 'location_notes', 'group', 'district', 'group_notes'],
-            array_keys($tsml_contact_fields)
-        );
+        $message .= '</p><table style="font:14px arial;width:100%;border-collapse:collapse;parray_keys($tsml_export_columns)adding:0;">';
+        $fields = array_keys($tsml_export_columns);
         foreach ($fields as $field) {
```

Comment:
i think this could be `foreach ($tsml_export_columns as $field => $field_name) {`. then you wouldn't need the changes above

Reply:
also you could delete the `$field_name = __(ucwords(str_replace('_', ' ', $field)), '12-step-meeting-list');` line below

Reply:
Refactoring lines 455-456 is fine, but L26 still needs $tsml_export_column else the email notif doesn't show any fields: 
![image](https://github.com/code4recovery/12-step-meeting-list/assets/81018718/41af254c-7503-4c84-aa93-61a47d3ca8bc)

Reply:
yah i just meant the change on line 454 - something weird got into the style